devel/ansible/roles/hotness-dev/files/config.toml

```diff
@@ -55,7 +55,7 @@ propagate = false
 handlers = ["console"]
 
 [log_config.root]
-level = "WARNING"
+level = "DEBUG"
```

Comment:
Was this only for debug ? or is it an intended change ?

Reply:
This is config used for local testing in Vagrant only, so this is intended change.